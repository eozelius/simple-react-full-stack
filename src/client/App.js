import React from 'react';
import './styles/app.scss';

const App = props => (
  <div>
    <h2>Hello world</h2>
    <p>
      props:
      {JSON.stringify(props)}
    </p>
  </div>
);

export default App;
