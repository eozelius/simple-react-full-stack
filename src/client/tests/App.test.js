import React from 'react';
import App from '../App';

describe(App, () => {
  test('it renders properly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
