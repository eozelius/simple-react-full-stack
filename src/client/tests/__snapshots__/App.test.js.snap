// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App it renders properly 1`] = `
<div>
  <h2>
    Hello world
  </h2>
  <p>
    props:
    {}
  </p>
</div>
`;
